R/assert.R

```diff
@@ -84,3 +93,85 @@ vec_is <- function(x, ptype = NULL, size = NULL) {
 
   TRUE
 }
+
+#' Is an object a vector?
+#'
+#' @description
+#'
+#' A vector is a collection of values. In practical terms, a vector is
+#' something that can be a column of a data frame. The opposite of a
```

Comment:
I think we need to say something about how this definition relates to the usual definition of a vector in R (i.e. an atomic vector or a list)